astronomy/orbital_elements_body.hpp

```diff
@@ -319,7 +346,9 @@ OrbitalElements::ToClassicalElements(
              : UnwindFrom(classical_elements.back().argument_of_periapsis, ω),
          classical_elements.empty()
              ? Mod(M, 2 * π * Radian)
-             : UnwindFrom(classical_elements.back().mean_anomaly, M)});
+             : UnwindFrom(classical_elements.back().mean_anomaly, M),
+         (1 - e) * equinoctial.a,
+         (1 + e) * equinoctial.a});
```

Comment:
Could we comment the name of the fields or use designated initializers in this list?  It's impossible to figure out what the various bits mean.